e2e/specs/st_multiselect.spec.js

```diff
@@ -15,6 +15,35 @@
  * limitations under the License.
  */
 
+ function selectNthOptionForKthMultiselectNoClose(n, k) {
+  cy.getIndexed(".stMultiSelect", k)
+    .find("input")
+    .click();
+  cy
+    .getIndexed("li", n)
+    .click();
+  // close the multiselect
+  cy.getIndexed(".stMultiSelect", k)
+    .find("input")
+    .click();
+}
+
+function selectNthOptionForKthMultiselectClose(n, k) {
+  cy.getIndexed(".stMultiSelect", k)
+    .find("input")
+    .click();
+  cy
+    .getIndexed("li", n)
+    .click();
+}
+
+function delNthOptionForKthMultiselect(n, k) {
+  cy.getIndexed('.stMultiSelect', k)
+    // (n + 1) * 2 - 1 because there are two spans: option text and x
+    .getIndexed('span[data-baseweb="tag"] span', (n + 1) * 2 - 1)
+    .click();
+}
```

Comment:
These helper functions are getting more and more unreadable/are beginning to duplicate one another. I think they could be written to be more sane and in a way that it's not necessary to also have comments explaining what the helper functions are doing when they're used.

Could we instead write something like

```javascript
function selectFromNthMultiselect(tag, n, closeAfterSelecting) {
  // select the value with text "tag" in the nth multiselect
  if (closeAfterSelecting) {
    // ...
  }
}

function delFromNthMultiselect(tag, n) {
  // ...
}
```

This way, their names are less verbose, it's easier to tell what's going on from the usage of the function itself, and we get to drop a helper function. I'd especially recommend selecting by text for the del helper function since computing indexes to be able to click on the right thing feels mildly absurd to me